silver/models/transactions/transaction.py

```diff
@@ -136,6 +136,43 @@ def refund(self, refund_code='default', refund_reason='Unknown refund reason'):
         self.refund_code = refund_code
         logger.error(str(refund_reason))
 
+    def save(self, *args, **kwargs):
+        previous_instance = get_object_or_None(Transaction, pk=self.pk) if self.pk else None
+        setattr(self, 'previous_instance', previous_instance)
+
+        def _clean_and_save():
+            if not getattr(self, '.cleaned', False):
```

Comment:
It looks like a method.
Do you use the closure, or you just want to reuse the code?